charts/sscs-case-loader/values.preview.template.yaml

```diff
@@ -0,0 +1,4 @@
+job:
+  image: ${IMAGE_NAME}
+  ingressHost: ${SERVICE_FQDN}
+  aadIdentityName:
```

Comment:
probably want to duplicate the vault definition here to remove the app insights key when you add it